src/components/time-input/TimeInput.scss

```diff
@@ -0,0 +1,9 @@
+@import '../../styles/variables';
+
+.bdl-TimeInput {
+    .bdl-TimeInput-icon {
+        display: inline-block;
+        margin-bottom: -3px;
+        margin-left: -24px;
```

Comment:
Assuming these have some relationship to hardcoded dimensions elsewhere. If not, can the value come from the grid units?

Reply:
Yup, these are based on the margins for the verified icon in https://github.com/box/box-ui-elements/blob/master/src/components/text-input/TextInput.scss#L16. The clock is a little taller than the verified icon, though, so the vertical margins have a one-pixel difference.